Tests/Console/Unit/Install/Upgrade/UpgradeWizardExecutorTest.php

```diff
@@ -88,4 +94,20 @@ public function wizardIsDoneButCalledWhenForced()
         $result = $subject->executeWizard('Foo\\Test', [], true);
         $this->assertFalse($result->hasPerformed());
     }
+
+    /**
+     * @param DummyUpgradeWizard|ObjectProphecy $upgradeWizardProphecy
+     */
+    private function assertOutputInitForChattyWizard(ObjectProphecy $upgradeWizardProphecy)
+    {
+        if (!interface_exists('TYPO3\\CMS\\Install\\Updates\\ChattyInterface')) {
+            return;
+        }
+
+        /** @var OutputInterface $outputInterfaceArgument */
+        $outputInterfaceArgument = Argument::type(OutputInterface::class);
+        /** @var MethodProphecy $setOutputMethod */
+        $setOutputMethod = $upgradeWizardProphecy->setOutput($outputInterfaceArgument);
+        $setOutputMethod->shouldBeCalled();
```

Comment:
This is a rather unusual way to write this, please merge these two lines.

Reply:
I wrote it like this to prevent PhpStorm warnings. Can you give me a hint how to write it in one line without IDE warnings? I have the [PHPUnit enhancement plugin](https://plugins.jetbrains.com/plugin/9674-phpunit-enhancement) installed.

Reply:
No idea about warnings, I (and everybody else) use `$mock->method()->shouldBeCalled()`.

Reply:
I wouldn't care much about this one. The whole thing needs to be reworked anyway, which most likely will obsolete this code as well.